command/command.c

```diff
@@ -112,7 +121,7 @@ command_about_callback (GtkAction *action, CommandApplet *command_applet)
                           "version", VERSION,
                           "copyright", copyright,
                           "authors", authors,
-                          "comments", _("Shows the output of a command"),
+                          "comments", _("Shows the output of a command and can launch an other"),
```

Comment:
Maybe change this to `"Shows the output of a command and can execute another one on click"`